src/App.tsx

```diff
@@ -1,14 +1,59 @@
-import { AppHeader } from "./components/AppHeader";
-import { AppLayout } from "./components/AppLayout";
+import { AppHeader } from './components/AppHeader';
+import { AppLayout } from './components/AppLayout';
+import './App.css';
+import React, { useState } from 'react';
+import { NewCard } from './components/NewCard';
+import { CardsList } from './components/CardsList';
+
+const initialCards = [
+	{ heading: 'tytul', answer: 'costamn', id: 1 },
+	{ heading: 'tytul', answer: 'costamn', id: 2 },
+];
+
 
-import "./App.css";
 
 function App() {
-  return (
-    <AppLayout>
-      <AppHeader cardsAmount={1}/>
-    </AppLayout>
-  );
+	const [newCard, setNewCard] = useState(false);
+	const [val, setVal] = useState(0);
+	const [cards, setCards] = useState(initialCards);
+
+	function createNewCard() {
+		{
+
+			setNewCard(true);
```

Comment:
**issue:** your prettier does not work for sure. We need to fix it, because its very hard to read now